ydb/docs/en/core/yql/reference/yql-core/syntax/create_view.md

```diff
@@ -0,0 +1,88 @@
+# CREATE VIEW
+
+CREATE VIEW defines a view of a query.
+
+A view is a logical representation of a table formed by the specified query. The view does not physically store the table, but runs the query to produce the data whenever the view is selected from.
```

Comment:
From my point of view, "result" is excessive in the "a table formed by a given query result", because it is not the result that forms the table, but the query. The query is the actor.

Reply:
Changed to:

"A view logically represents a table formed by a given query. The view does not physically store the table but executes the query to produce the data whenever the view is accessed."